CHANGELOG.mdx

```diff
@@ -144,8 +251,12 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 - [#96](https://github.com/rasahq/rasa/issues/96): Update the `LanguageModelFeaturizer` tests to reflect new default model weights for `bert`, and skip all `bert` tests 
   with default model weights on CI, run `bert` tests with `bert-base-uncased` on CI instead.
 
+### Improved Documentation
```

Comment:
@Tawakalt @mprazz These are old ones which were omitted in the past due to wrong names (`docs` instead `doc` and `mis` instead of `misc`